tuf/client/updater.py

```diff
@@ -1385,7 +1385,7 @@ def _verify_root_self_signed(self, signable):
     signatures = signable['signatures']
     signed = securesystemslib.formats.encode_canonical(
         signable['signed']).encode('utf-8')
-    validated = 0
+    verified_sig_keyids = []
```

Comment:
Might as well just use a set() from the outset, no?

Reply:
That's still a list. Proper empty set requires `verified_sig_keyids = set()` (and then you need to use use `verified_sig_keyids.add(keyid)`, but don't need to transform it before getting the length).

Either way works for me, maybe it's cleaner to be an actual set to begin with -- this is also what `sig.verify()` does.

Reply:
Quite right, thanks both. Pushed a fixup to change this to a set in 44d48af

Reply:
oh and if my comment sounded weird that was because I misread the trishanks comment to be a github 'suggestion'. Didn't mean to lecture :)